viewing_party/party.py

```diff
@@ -1,23 +1,161 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating :
+        return None 
+    else:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    
+    
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data 
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data 
+
+def watch_movie(user_data,title):
+
+    for movie in user_data["watchlist"] :
+
+        if title in movie["title"] :

```

Comment:
The spacing in this function is a little weird. Blank lines should be used to meaningfully separate code. Since the loop code here is all tightly related to each other, I would consider removing these lines since they don't signify a meaningful separation from code that's performing another task. 

```suggestion
def watch_movie(user_data,title):
    for movie in user_data["watchlist"] :
        if title in movie["title"] :
```